### ISSUE TITLE
Accessibility on the buttons

### ISSUE THREAD

**Issue by yikman** [wojtekmaj/react-calendar#125]
Hello,

When using voiceover on the calendar, all it says 'button'.
Is it possible to add aria label onto each button to specify what the button is?

Thanks

**Comment by wojtekmaj** [comment]
Well this sucks. Each button contains `<time>` element which should be understood by voiceover. There's no possibility of adding aria-label now, and I need to figure out something to make it work in all languages by default.

**Comment by yikman** [comment]
Voiceover doesn't pick up the `<time>` just says button.

**Comment by hugotox** [comment]
Hello! 
This is still an issue. `abbr` is not an interactive element, so it doesn't support `aria-label`. `aria-label` should be applied to the button instead
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/1758062/178376004-0b22e72c-c524-4a61-a35f-5dad7a704234.png">

**Comment by rodriguezmarting** [comment]
Agreed, the bug is still happening on `v3.0.0`

**Comment by wojtekmaj** [comment]
"Still"? 3.0.0 has been released 3 years ago.
